packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.tsx

```diff
@@ -17,7 +17,9 @@ const MultiInputDateRangeFieldRoot = styled(
     slot: 'Root',
     overridesResolver: (props, styles) => styles.root,
   },
-)({});
+)({
+  alignItems: 'baseline',
```

Comment:
Not related, the v5 pickers had this alignment and I find it slightly better

Reply:
Was it an oversight to apply this change only to this multi input field?
Also, I should probably be done via the `stack` prop not to have overridden style rules. 🤔 
![Screenshot 2022-12-16 at 15 59 12](https://user-images.githubusercontent.com/4941090/208114443-a360b10a-a8d7-48ce-bda6-7b56828045fc.png)

Reply:
You are probably right on both points yes

Reply:
Will address it with Multi Input fields PR. 😉